### PR TITLE
[doc] Update git-subtree-check.sh README

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: contrib/devtools/README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ maintained:
 * for `src/secp256k1`: https://github.com/bitcoin-core/secp256k1.git (branch master)
 * for `src/leveldb`: https://github.com/bitcoin-core/leveldb.git (branch bitcoin-fork)
 * for `src/univalue`: https://github.com/bitcoin-core/univalue.git (branch master)
+* for `src/crypto/ctaes`: https://github.com/bitcoin-core/ctaes.git (branch master)
 
 Usage: `git-subtree-check.sh DIR COMMIT`
```
Comment (laanwj) [inline contrib/devtools/README.md]: Could also update this, with that the COMMIT argument is optional.

Comment (MarcoFalke) [inline contrib/devtools/README.md]: Check the line below, which says

```markdown
`COMMIT` may be omitted, in which case `HEAD` is used.

**Comment by laanwj** [comment]
Thanks, ACK

**Comment by MarcoFalke** [comment]
Trivial PR. Does this require more review?

**Pull request merged**